### PR TITLE
API setup with giraffe

### PR DESCRIPTION
**Pull request by bogdangd-zz**

### CODE REVIEW

File: Book-A-Desk.Api/Program.fs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿open Microsoft.AspNetCore.Builder
+open Microsoft.AspNetCore.Hosting
+open Microsoft.Extensions.Hosting
+open Microsoft.Extensions.DependencyInjection
+open Giraffe
+
+open BookADesk.Api.HttpHandlers
+
+let webApp =
```
Comment (TomasMorton) [inline Book-A-Desk.Api/Program.fs]: Personally, I prefer routing to be in a different file to the .NET setup.

**Review by TomasMorton — DISMISSED** [review]

File: Book-A-Desk.Api/Program.fs (hunk 1 above)
Comment (bogdangd-zz) [reply Book-A-Desk.Api/Program.fs]: @TomasMorton I extracted the routes to a separate file. Can you please review again?

**Review by TomasMorton — APPROVED** [review]
LGTM, but you might want an F# Guru to look :)

**Review by StSenff — APPROVED** [review]
Looks good enough for a first implementation

**Pull request merged**